text/0000-io-error-handling.md

```diff
@@ -0,0 +1,81 @@
+- Feature Name: my_awesome_io_error_handling
```

Comment:
Please make a reasonable suggestion, there's no way this would be introduced under `#![feature(my_awesome_io_error_handling)]`.